specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -0,0 +1,2661 @@
+              "description": "Specifies HSM settings of the amlFilesystem."
+            },
+            "namespaceHydrationStatus": {
+              "description" : "Namespace Hydration Status during creation",
+              "$ref": "#/definitions/amlFilesystemNamespaceHydrationStatus"
+            },
+            "archiveSettings" : {
+              "description": "Archive settings and status",
+              "$ref" :  "#/definitions/amlFilesystemArchive"
+            }
+          },
+          "type": "object"
+        },
+        "sku": {
+          "description": "SKU for the Cache.",
+          "properties": {
+            "name": {
+              "description": "SKU name for this Cache.",
+              "type": "string"
+            }
+          },
+          "type": "object"
+        }
+      },
+      "x-ms-azure-resource": true,
+      "type": "object"
+    },
+    "amlFilesystemName" : {
+      "description": "Name for the amlFilesystem.  Alphanumerics, underscores, and hyphens. Start and end with alphanumeric.",
+      "type" : "string",
+      "pattern": "^[0-9a-zA-Z][-0-9a-zA-Z_]{0,78}[0-9a-zA-Z]$",
+      "minLength": 2,
+      "maxLength": 80
+    },
+    "amlFilesystemsListResult": {
+      "description": "Result of the request to list Caches. It contains a list of Caches and a URL link to get the next set of results.",
+      "properties": {
+        "nextLink": {
+          "description": "URL to get the next set of Cache list results, if there are any.",
+          "type": "string"
+        },
+        "value": {
+          "description": "List of Caches.",
+          "items": {
+            "$ref": "#/definitions/amlFilesystem"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "amlFilesystemIdentity": {
+      "description": "amlFilesystem identity properties.",
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal id of the amlFilesystem."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant id associated with the amlFilesystem."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of identity used for the amlFilesystem",
+          "enum": [
+            "SystemAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "amlFilesystemIdentityType",
+            "modelAsString": false
+          }
+        }
+      }
+    },
+    "amlFilesystemMaintenanceStatus": {
+      "description": "Properties describing the software upgrade state of the amlFilesystem.",
+      "properties": {
+        "currentSoftwareVersion": {
+          "description": "Version string of the software currently installed on this amlFilesystem.",
+          "type": "string",
+          "readOnly": true
+        },
+        "softwareUpdateStatus": {
```

Comment:
AML should not have this softwareUpdateStatus. There is nothing the customer can do about it.

Reply:
removed.